test/function_tests/Uart/test_Uart_write.c

```diff
@@ -0,0 +1,22 @@
+#include <stdbool.h>
+#include "Uart.h"
+
+static int timeoutLimit = 2000000;
+
+static Uart_ErrorCode errCode = Uart_ErrorCode_OK;
+
+static uint8_t data[] = "Write text (sync)\r\n";
+
+bool
+test_Uart_write(Uart* uart)
+{
+    uart->txFifo = NULL;
```

Comment:
Why is this needed?

Reply:
It's not needed in this scenario but if this test will be used in different order then not disabling the fifo will return false failure. I think it isn't good practice to assume hard requirement of order in which tests should be performed if it's not necessary.

Reply:
See my comment regarding test_Uart_read